[ECOINT-289][Bugfix] Add back test cases with fixed sr (#2818)
DataDog/integrations-extras: celerdata/CHANGELOG.md

# CHANGELOG - celerdata

## 1.2.1 / 2025-10-01

***Fixed***:

> * Since StarRocks FE has fixed the metrics format issue, now it is needed to restore the deleted test case.

Reviewer: ```suggestion
* Restored the previously removed test case, as the metrics format issue in StarRocks FE has been resolved
```
